test/document_stores/test_elasticsearch.py

```diff
@@ -294,6 +302,7 @@ def test_elasticsearch_brownfield_support(self, ds, documents):
         assert len(transferred_documents) == len(set(" ".join(original_content).split()))
 
     @pytest.mark.unit
+    @pytest.mark.skipif(ES_VERSION[0] != 7, reason="aws4auth only supported in ES7.")
```

Comment:
Same, why not test it in `test_elasticsearch7.py`?

Reply:
True, I'll move this.